src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/preferences.html.twig

```diff
@@ -37,144 +37,37 @@
-                  <div class="col-sm">
-                    {{ form_errors(generalForm.display_manufacturers) }}
-                    {{ form_widget(generalForm.display_manufacturers) }}
-                    <small class="form-text">{{ 'Enable brands page on your front office even when its module is disabled.'|trans({}, 'Admin.Shopparameters.Help') }}</small>
-                  </div>
-                </div>
-                <div class="form-group row">
-                  <label class="form-control-label">{{ 'Display best sellers'|trans }}</label>
-                  <div class="col-sm">
-                    {{ form_errors(generalForm.display_best_sellers) }}
-                    {{ form_widget(generalForm.display_best_sellers) }}
-                    <small class="form-text">{{ 'Enable best sellers page on your front office even when its respective module is disabled.'|trans({}, 'Admin.Shopparameters.Help') }}</small>
-                  </div>
-                </div>
-                <div class="form-group row">
-                  <label class="form-control-label">{{ 'Enable Multistore'|trans }}</label>
-                  <div class="col-sm">
-                    {{ form_errors(generalForm.multishop_feature_active) }}
-                    {{ form_widget(generalForm.multishop_feature_active) }}
-                    <small class="form-text">{{ 'The multistore feature allows you to manage several e-shops with one Back Office. If this feature is enabled, a "Multistore" page will be available in the "Advanced Parameters" menu.'|trans({}, 'Admin.Shopparameters.Help') }}</small>
-
-                    {% if generalForm.multishop_feature_active.vars.disabled %}
-                      <div class="alert alert-warning mt-2 mb-0" role="alert">
-                        <p class="alert-text">
-                          {{ 'You can\'t change the value of this configuration field in the context of this shop.'|trans({}, 'Admin.Notifications.Warning') }}
-                        </p>
-                      </div>
-                    {% endif %}
-                  </div>
-                </div>
-                <div class="form-group row">
-                  <label class="form-control-label">{{ 'Main Shop Activity'|trans }}</label>
-                  <div class="col-sm">
-                    {{ form_errors(generalForm.shop_activity) }}
-                    {{ form_widget(generalForm.shop_activity) }}
-                  </div>
-                </div>
-                {% block shop_preferences_form_rest %}
-                  {{ form_rest(generalForm) }}
-                {% endblock %}
+                {{ form_widget(generalForm) }}
               </div>
             </div>
             <div class="card-footer">
               <div class="d-flex justify-content-end">
                 <button class="btn btn-primary" id="form-preferences-save-button">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
+                {{ form_widget(generalForm) }}
```

Comment:
Is it intended to have `form_widget(generalForm)` twice? 🤔

Reply:
Not it wasn't. Fixed it!